developers/bindings/thing-xml.md

```diff
@@ -509,17 +509,36 @@ The `representation property` will be defined in the thing type XML:
     </thing-type>
 ```
 
-Note that the `representation property` is normally not listed in the `properties` part of the thing type, as this part contains static properties, that are the same for each thing of this thing type.
-The name of the `representation property` identifies a property that is added to the thing in the thing handler upon successful initialization.
+Note that the `representation property` is normally not listed in the `properties` part of the thing type XML, as this part contains static properties, that are the same for all instances of this thing type.
+The name of the `representation property` identifies a property or configuration parameter that is added to the thing in the thing handler upon successful initialization.
 
 ### Representation Property and Discovery
 
-The representation property is being used to auto-ignore discovery results of devices that already have a corresponding thing.
-This happens if a device is being added manually.
-If the new thing is going online, the auto-ignore service of the inbox checks if the inbox already contains a discovery result of the same type where the value of its `representation property` is identical to the value of the `representation property` of the newly added thing.
-If this is the case, the result in the inbox is automatically set to ignored.
-Note that this result is automatically removed when the manual added thing is eventually removed.
-A new discovery would then automatically find this device again and add it to the inbox properly.
+The representation property is used to auto-ignore discovery results of things that already exist in the system.
+This can happen a) if a thing has been created manually, or b) it has been discovered separately by two mechanisms e.g. by mDNS, and by NetBios, or UPnP.
```

Comment:
`This can happen if, a) a Thing...`

`or b) the Thing has been discovered...`

`e.g. by mDNS, NetBios, or UPnP.`

Reply:
Ok. Done.